cpp/memilio/math/stepper_wrapper.h

```diff
@@ -69,13 +69,13 @@ class ControlledStepperWrapper : public mio::IntegratorCore<FP>
     }
 
     /**
-    * @brief Make a single integration step on a system of ODEs and adapt the step size dt.
-
-    * @param[in] yt Value of y at t_{k}, y(t_{k}).
-    * @param[in,out] t Current time step t_{k} for some k. Will be set to t_{k+1} in [t_{k} + dt_min, t_{k} + dt].
-    * @param[in,out] dt Current time step size h=dt. Overwritten by an estimated optimal step size for the next step.
-    * @param[out] ytp1 The approximated value of y(t_{k+1}).
-    */
+     * @brief Make a single integration step on a system of ODEs and adapt the step size dt.
+     *
+     * @param[in] yt Value of y at t_{k}, y(t_{k}).
+     * @param[in,out] t Current time step t_{k} for some k. Will be set to t_{k+1} in [t_{k} + dt_min, t_{k} + dt].

```

Comment:
This is a bit unnecessarily complex in my opinion  :)
And I'm not quite clear about the "time step" in the first sentence.

Reply:
I was thinking of the steps t0, t1, ... but I can remove the word step, it is not necessary here.
The k helps with formulating what ytp1 is supposed to mean (especially the p1 "plus one" part). I would also be fine introducing a t' (or s?) instead of the whole k business.